datafusion/src/logical_plan/plan.rs

```diff
@@ -212,6 +212,15 @@ pub enum LogicalPlan {
         /// The logical plan
         input: Arc<LogicalPlan>,
     },
+    /// Drops an in memory table.
+    DropMemoryTable {
```

Comment:
I think it should be called `DropTable` as we are also dropping external tables.